translations/monero-core_eo.ts

```diff
@@ -1238,16 +1394,11 @@
     <name>TickDelegate</name>
     <message>
         <location filename="../components/TickDelegate.qml" line="55"/>
-        <source>Normal</source>
-        <translation>Normala</translation>
+        <source>Default</source>
```

Comment:
Cool, I have integrated changes from https://github.com/monero-project/monero-gui/commit/21a9bfb5e4c7b7c1bbfe4fb1b97c8714847c41d1 and updated translations.